### PR TITLE
add(ui-libraries): Tail-kit

### PR DESCRIPTION
**Pull request by Charlie85270**
| Name                 | Link                 |
| -------------------- | -------------------- |
| Tail-kit | https://tailwind-kit.com |

Tail-kit, a free and open source components Kit based on Tailwind CSS 2.0.

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] I have read and followed the [contribution guidelines](.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md
Comment (innocenzi) [inline README.md:233]:
```suggestion
- 🧩 [Tail-kit](https://tailwind-kit.com/) - 200+ free and open-source components.
```

**Review by innocenzi — COMMENTED** [review]
Related to the name (I see `Tailwind UI Kit` in the title of your site), I'd like to get @joshmanders' approval before merging this.

**Comment by tonyketcham** [comment]
This looks fantastic

**Comment by joshmanders** [comment]
I've consulted with Adam about the naming of this project, once his verdict comes back I'll update.

File: README.md
Comment (joshmanders) [inline README.md:233]: After talking with Adam, the domain still violates brand usage guidelines. Once that is resolved and we don't see any violations of the [Tailwind Brand Usage Guidelines](https://tailwindcss.com/brand#usage), we'll happily accept the PR.

**Review by joshmanders — CHANGES_REQUESTED** [review]

**Comment by joshmanders** [comment]
Closing as no response in 10 days.

**Pull request closed without merging**
